docs/gitbook/key-concepts/chain_joins.adoc

```diff
@@ -0,0 +1,51 @@
+= Chain Joins
+
+Chain joins are an Interbit-specific feature. They permit some (or all)
+of the information in one blockchain to be connected with information in
+another blockchain. These connections are controlled via ACLs, and so
+are explicit and purposeful.
+
+There are two kinds of chain joins: read joins and write joins.
+
+== Read Joins
+
+image:/architecture/img/read_join.svg["An Interbit read join between a
+provider blockchain and a consumer blockchain", 478, 111, role="center"]
+
+A read join means that one blockchain, the _consumer_, has access to a
+copy of a portion of the state -- the collection of data stored on the
+blockchain -- of another blockchain, the _provider_. Any updates to the
+provider's state are automatically updated in the consumer's blockchain.
```

Comment:
It might be worth mentioning precisely when the automatic update occurs.

Reply:
Can you elaborate on the conditions/events that prompt auto-update?

Reply:
@pekeler  Hi Christian, can you describe how the automatic updates propagate?

Reply:
Welp I don't think this should hold up merging this so I guess we can ignore this for now.